app/posts/manage-teacher-training-applications/2021-04-08-gathering-school-direct-contact-details-from-higher-education-institutions.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Gathering school direct contact details from higher education institutions
+description: Using a spreadsheet to help higher education institutions invite their school direct partners to use  Manage
+date: 2021-04-08
+---
+
+We’ve been looking at ways to help higher education institutions (HEIs) to:
+
+- set themselves up to use Manage
+- invite their school direct partners (SDs) to set themselves up
+
+We started by designing a [small online service that allowed HEIs to set themselves up and invite SDs](https://bat-design-history.netlify.app/manage-teacher-training-applications/self-service-registration/). 
+
+After testing that design we decided to explore a simpler approach, focusing on gathering contact details for SDs.
+
+## What we did and why
+
+We created a spreadsheet template that we could send to HEIs as a CSV file. The columns in the template were:
```

Comment:
Do we need to mention why we chose a CSV over Excel, OpenDocument or Google Sheet, or is this implicit? I think it'd help others to explain the choice. I know you mention a little later the option of a Google Sheet, but it could be useful to say why not Excel or OpenDocument.